host/xtest/xtest_helpers.c

```diff
@@ -408,3 +409,44 @@ int ree_fs_get_ta_dirname(TEEC_UUID *p_uuid, char *buffer, uint32_t len)
 			p_uuid->clockSeqAndNode[6],
 			p_uuid->clockSeqAndNode[7]);
 }
+
+/* Misc auxilary functions */
+void tee_errx(const char *msg, TEEC_Result res)
+{
+	fprintf(stderr, "%s: 0x%08x", msg, res);
```

Comment:
Forget this about 'error origin'. does not apply to all cases where this routine can be called.

Minor change: add `\n` at string end. it was missing in the original routine.